sdk/codesigning/Azure.ResourceManager.CodeSigning/src/Generated/CodeSigningCertificateProfileData.cs

```diff
@@ -0,0 +1,130 @@
+            OrganizationUnit = organizationUnit;
+            StreetAddress = streetAddress;
+            DoesIncludeStreetAddress = doesIncludeStreetAddress;
+            State = state;
+            DoesIncludeState = doesIncludeState;
+            City = city;
+            DoesIncludeCity = doesIncludeCity;
+            PostalCode = postalCode;
+            DoesIncludePostalCode = doesIncludePostalCode;
+            Country = country;
+            DoesIncludeCountry = doesIncludeCountry;
+            EnhancedKeyUsage = enhancedKeyUsage;
+            IdentityValidationId = identityValidationId;
+            ProvisioningState = provisioningState;
+            Status = status;
+            Certificates = certificates;
+        }
+
+        /// <summary> Profile type of the certificate. </summary>
+        public CertificateProfileType ProfileType { get; set; }
+        /// <summary> Rotation policy of the certificate. </summary>
+        public CertificateRotationPolicy RotationPolicy { get; set; }
+        /// <summary> Used as CN in the subject name of the certificate. </summary>
+        public string CommonName { get; set; }
+        /// <summary> Used as O in the subject name of the certificate. </summary>
+        public string Organization { get; set; }
+        /// <summary> Used as OU in the subject name of the private trust certificate. </summary>
+        public string OrganizationUnit { get; set; }
+        /// <summary> Gets the street address. </summary>
+        public string StreetAddress { get; }
+        /// <summary> Whether to include in the public trust certificate subject name. </summary>
+        public bool? DoesIncludeStreetAddress { get; set; }
+        /// <summary> Gets the state. </summary>
+        public string State { get; }
+        /// <summary> Whether to include in the public trust or private trust certificate subject name. </summary>
+        public bool? DoesIncludeState { get; set; }
+        /// <summary> Gets the city. </summary>
+        public string City { get; }
+        /// <summary> Whether to include in the public trust or private trust certificate subject name. </summary>
+        public bool? DoesIncludeCity { get; set; }
+        /// <summary> Gets the postal code. </summary>
+        public string PostalCode { get; }
+        /// <summary> Whether to include in the public trust certificate subject name. </summary>
+        public bool? DoesIncludePostalCode { get; set; }
+        /// <summary> Gets the country. </summary>
+        public string Country { get; }
+        /// <summary> Whether to include in the public trust or private trust certificate subject name. </summary>
+        public bool? DoesIncludeCountry { get; set; }
+        /// <summary> Enhanced key usage of the certificate. </summary>
+        public string EnhancedKeyUsage { get; }
+        /// <summary> Gets the identity validation id. </summary>
+        public string IdentityValidationId { get; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).